### DISCUSSION TITLE
Welcome to nfpm Discussions!

### DISCUSSION THREAD

**Discussion by caarlos0** [goreleaser/nfpm#326] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jon-hotaisle** [comment]
Wow, let's get this party started after 4 years. Thanks for building this, it looks perfect. I used fpm many years ago, so having something with zero dependencies is very nice.

My name is Jon Stevens, I'm the CEO of Hot Aisle and I'm looking to integrate this solution into my CI/CD for a little golang [CLI tool](https://github.com/hotaisle/hotaisle-cli) I'm building for my business.

One bit of feedback that I'm seeing off the bat is that in the docs, it says that templating isn't supported, but it isn't really clear what that means? If I want to build for all the different arch's... isn't ${ARCH} a template variable? Or is that just an environment variable? I guess I'll have to dig in and figure this out, but it would be nice if the documentation was a bit more clear. Ideally, I could just run nfpm one time and have it output all the various arch / packaging solutions in one go (run it internally as goroutines for max speed). Instead, it looks like I have to run it a bunch of times and pass different environment variables to my yaml.

**Reply by caarlos0** [reply]
it doesn't support templating, but some fields might support environment variable substitution.

check https://nfpm.goreleaser.com/docs/configuration/ for more info :)
